src/raw-value.ts

```diff
@@ -126,64 +78,137 @@ function createReverseShadowTarget(target: ReverseProxyTarget): ReverseShadowTar
     return shadowTarget;
 }
 
+// equivalent to Object.getOwnPropertyDescriptor, but looks into the whole proto chain
+function getPropertyDescriptor(o: any, p: PropertyKey): PropertyDescriptor | undefined {
+    do {
+        const d = ReflectGetOwnPropertyDescriptor(o, p);
+        if (!isUndefined(d)) {
+            ReflectSetPrototypeOf(d, null);
+            return d;
+        }
+        o = ReflectGetPrototypeOf(o);
+    } while (o !== null);
+    return undefined;
+}
+
 export function reverseProxyFactory(env: MembraneBroker) {
 
-    function getReverseDescriptor(descriptor: PropertyDescriptor): PropertyDescriptor {
-        const reverseDescriptor = assign(ObjectCreate(null), descriptor);
-        const { value, get, set } = reverseDescriptor;
-        if ('writable' in reverseDescriptor) {
+    function getRawDescriptor(secDesc: PropertyDescriptor): PropertyDescriptor {
+        const rawDesc = assign(ObjectCreate(null), secDesc);
+        const { value, get, set } = rawDesc;
+        if ('writable' in rawDesc) {
             // we are dealing with a value descriptor
-            reverseDescriptor.value = isFunction(value) ?
+            rawDesc.value = isFunction(value) ?
                 // we are dealing with a method (optimization)
                 getRawFunction(value) : getRawValue(value);
         } else {
             // we are dealing with accessors
             if (isFunction(set)) {
```

Comment:
For the secure descriptor helper we don't do the function check for `get` and `set` but in the raw descriptor helper we do. Can we relax the raw descriptor helper to that of the secure helper in that respect?